google-scripts/process-form-submission/__tests__/setUniqueIdOnSubmission.test.ts

```diff
@@ -65,7 +71,7 @@ describe("#setUniqueIdOnSubmission", () => {
       mockEvent
     );
 
-    expect(response).toBe(2);
+    expect(response).toStrictEqual({ id: 2, row: 7 });
```

Comment:
I decided on a name change rather than a separate test as in my head I can't think of there being much value in having one test only test the row number is retrieved and one test for the complete object. Guess we could split it into three tests or have multiple expectations within the one test but I think changing the name is more suitable.

Happy to hear others thoughts though and change it if required!